### DISCUSSION TITLE
3D extension

### DISCUSSION THREAD

**Discussion by lancu** [cvat-ai/cvat#2790] [General]
Hi @manasars  Are there any instructions on how to install/run the 3D extension for CVAT ? 
Tried to use it out of the box, however I cannot load any PCD file in a new task 
Thank you!

**Comment by bsekachev** [comment]
Hi, @lancu 

3D is not ready yet to usage, but it is expected that CVAT is able to create a task using .zip archives with PCD files. 
What problems did you meet?

**Reply by lancu** [reply]
Thanks @bsekachev, 
I am aware it's not ready I'm was testing the task creation with some own files. Didn't realize one needs to upload as "zip" and was uploading single pcd/bin files. 
Works all fine when uploading ziped collections of pcds.
